gammapy/utils/interpolation.py

```diff
@@ -182,6 +186,22 @@ def _inverse(self, values):
         return np.power(values, 2)
 
 
+class StatProfileScale(InterpolationScale):
```

Comment:
Yes, I think it helps for to smooth the color transitions:

## Linear scaling
![Screenshot 2021-08-03 at 09 23 58](https://user-images.githubusercontent.com/3715928/127975130-5b9af6ef-9205-46c9-8a46-e09e0265c36c.png)

##  Stat profile scaling
![Screenshot 2021-08-03 at 09 24 43](https://user-images.githubusercontent.com/3715928/127975093-356e2164-39e6-49c8-b723-6b2c0879e9e1.png)

Side comment: the fact that the scaling was available as one of the options (instead of a separate helper method), made the comparison very easy...another minor argument for keeping it.